src/security/src/main/kotlin/org/vaccineimpact/api/security/WebTokenHelper.kt

```diff
@@ -74,8 +74,11 @@ open class WebTokenHelper(
 
     private fun modelReviewClaims(user: InternalUser): Map<String, Any>
     {
-        val modelsToReview = getReviewersMap()[user.username]?.map { it to "true" }
-                ?.toMap() ?: mapOf()
+        val modelsToReview = getReviewersMap()[user.username]?: listOf()
+        val groupPermissions = (user.roles.filter { it.name == "member" }
+                .map{ it.scope.databaseScopeId } + modelsToReview)
```

Comment:
I wondered if there was any circumstances in which you might want to distinguish the groups which a user is a member of vs the groups they are a reviewer of. But there's no need for that in our current use case since both members and reviewers should be able to see all docs for their groups (if modellers wanted to keep docs private to themselves they wouldn't be sharing them with us!)